lexos/static/js/scripts_cut.js

```diff
@@ -1,382 +1,663 @@
+            value="words"/>Tokens/Segment</label></div></div><div \
+            class="row cutting-radio"><div class="col-md-5"><label \
+            class="radio sizeradio"><input type="radio" \
+            name="cutType_' + fileID + '" id="cutTypeIndLines_' + fileID + '" \
+            value="lines"/>Lines/Segment</label></div><div \
+            class="col-md-7"><label class="radio numberradio"><input \
+            type="radio" name="cutType_' + fileID + '" \
+            id="cutTypeIndNumber_' + fileID + '" \
+            value="number"/>Segments/Document</label></div></div></div><div \
+            class="row"><div class="col-md-6 pull-right" \
+            style="padding-left:2px;padding-right:3%;"><label><span \
+            id="numOf' + fileID + '" class="cut-label-text">Number of \
+            Segments:</span><input type="number" min="1" step="1" \
+            name="cutValue_' + fileID + '" class="cut-text-input" \
+            id="individualCutValue" value=""/></label></div></div><div \
+            class="row overlap-div"><div class="col-md-6 pull-right" \
+            style="padding-left:2px;padding-right:3%;"><label>Overlap: \
+            <input type="number" min="0" name="cutOverlap_' + fileID + '" \
+            class="cut-text-input overlap-input" id="individualOverlap" \
+            value="0"/></label></div></div>\
+            <div id="lastprop-div_' + fileID + '" \
+            class="row lastprop-div"><div class="col-md-6 pull-right" \
+            style="padding-left:2px;padding-right:1%;"><label>Last Proportion \
+            Threshold: <input type="number" min="0" \
+            id="cutLastProp_' + fileID + '" \
+            name="cutLastProp_' + fileID + '" \
+            class="cut-text-input lastprop-input" value="50" \
+            style="width:54px;margin-right:3px;"/> %</label></div></div><div \
+            class="row"><div class="col-md-6 pull-right" \
+            style="padding-left:2px;padding-right:1%;">\
+            <label>Cutset Label: <input type="text" \
+            name="cutsetnaming_' + fileID + '" class="cutsetnaming" \
+            value="' + filename + '" style="width:155px;display:inline; \
+            margin: auto; white-space: nowrap; overflow: hidden; \
+            text-overflow: ellipsis;></label></div></div><div \
+            class="row cuttingdiv" id="cutByMSdiv"><div \
+            class="col-sm-4"><label><input type="checkbox" \
+            class="indivMS" name="cutByMS_' + fileID + '" \
+            id="cutByMS_' + fileID + '"/>Cut by Milestone</label></div><div \
+            class="col-sm-8 pull-right" id="MSoptspan" \
+            style="display:none;"><span>Cut document on this term \
+            <input type="text" class="indivMSinput" \
+            name="MScutWord_' + fileID + '" id="MScutWord' + fileID + '" \
+            value="" \
+            style="margin-left:3px;width:130px;"/></span></div>\
+            </div></fieldset></div>'
```

Comment:
Holy shit, this is intense. Also, template literal, and make it look like a real html, instead of some messed up lines of crappy strings.